src/App.tsx

```diff
@@ -15,36 +16,102 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  ALPHABET = 'alphabet',
+  LENGTH = 'length',
+  DEFAULT = '',
+}
+
+function getPreparedGoods(
+  goods: string[],
+  sortField: SortType,
+  reverse: boolean,
+): string[] {
+  const preparedGoods = [...goods];
+
+  if (sortField) {
+    preparedGoods.sort((good1, good2) => {
+      switch (sortField) {
+        case SortType.ALPHABET:
+          return good1.localeCompare(good2);
+
+        case SortType.LENGTH:
+          return good1.length - good2.length;
+
+        default:
+          return 0;
+      }
+    });
+  }
+
+  if (reverse) {
+    preparedGoods.reverse();
+  }
+
+  return preparedGoods;
+}
+
 export const App: React.FC = () => {
+  const [currentSort, setCurrentSort] = useState<SortType>(SortType.DEFAULT);
+  const [reverse, setReverse] = useState<boolean>(false);
```

Comment:
```suggestion
  const [isReverse, setIsReverse] = useState<boolean>(false);
```
A great practice is to start boolean state names from `is`